app/data_model/submitted_responses.py

```diff
@@ -0,0 +1,81 @@
+import boto3
+from botocore.exceptions import BotoCoreError, ConnectionError as BotoConnectionError
+from app.settings import EQ_ENVIRONMENT_PREFIX, EQ_SUBMISSION_VALIDITY, EQ_DYNAMODB_ENDPOINT
+from structlog import get_logger
+
+logger = get_logger()
+
+
+class SubmittedResponses(object):
+
+    def __init__(self):
+        if EQ_DYNAMODB_ENDPOINT:
```

Comment:
if `EQ_DYNAMODB_ENDPOINT` defaults to None, what happens if you pass None into the `endpoint_url`?
ie, do you need the if/else?